install-deps.sh

```diff
@@ -306,9 +306,11 @@ else
     [ $WITH_ZBD ] && with_zbd=true || with_zbd=false
     [ $WITH_PMEM ] && with_pmem=true || with_pmem=false
     source /etc/os-release
+    pip install jsondiff
```

Comment:
Provided `jsondiff` is added in `requirements.txt`, I'm not sure we really need this.

Reply:
I see a `python3-jsondiff` package for Ubuntu and Fedora, but since not available in CentOS, let's install from requirements.